qa-physics/monitorRead.groovy

```diff
@@ -119,6 +121,13 @@ else if(RG=="RGF") {
   else if(runnum>=12444 && runnum<=12951) EBEAM = 10.389 // RCDB may still be incorrect
   else System.err << "ERROR: unknown beam energy\n"
 }
+else if(RG=="RGM") {
+  if     (runnum>=15013 && runnum<=15490) EBEAM = 5.98636 // RCDB may still be incorrect
+  else if(runnum>=15533 && runnum<=15727) EBEAM = 2.07052 // RCDB may still be incorrect
+  else if(runnum>=15728 && runnum<=15784) EBEAM = 4.02962 // RCDB may still be incorrect
+  else if(runnum>=15787 && runnum<=15884) EBEAM = 5.98636 // RCDB may still be incorrect
+  else System.err << "ERROR: unknown beam energy\n"
+}
```

Comment:
Is RCDB incorrect for RGM too? Otherwise we should remove the comments:
```suggestion
else if(RG=="RGM") {
  if     (runnum>=15013 && runnum<=15490) EBEAM = 5.98636
  else if(runnum>=15533 && runnum<=15727) EBEAM = 2.07052
  else if(runnum>=15728 && runnum<=15784) EBEAM = 4.02962
  else if(runnum>=15787 && runnum<=15884) EBEAM = 5.98636
  else System.err << "ERROR: unknown beam energy\n"
}
```